package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "reservations",
+  "version": "1.0.0",
+  "description": "reservation module for mock up Open Table page",
+  "main": "./server/index.js",
+  "scripts": {
+    "test": "node_modules/jest/bin/jest.js --watch",
+    "start": "nodemon -w ./server/index.js",
```

Comment:
@LittleJennie 
@cminho921 

From Nodemon's documentation:

By default nodemon monitors the current working directory. If you want to take control of that option, use the --watch option to add specific paths:

nodemon --watch app --watch libs app/server.js

Now nodemon will only restart if there are changes in the ./app or ./libs directory. By default nodemon will traverse sub-directories, so there's no need in explicitly including sub-directories.

Is this a purposeful change to only watch the server/index file for changes vs. any changes to any .js files in your working directory?

Reply:
I didn't quite understand the 'watch' command here at first. The '-w' command is later removed in the 4th PR.